### ISSUE TITLE
The item does not appear in the Windows 11 context menu

### ISSUE THREAD

**Issue by cleuton4k** [ubuntuegor/FluentInfo#1]
<img width="737" alt="ubuntuegor_FluentInfo_ See detailed media information in a modern fashion - Context menu option" src="https://github.com/ubuntuegor/FluentInfo/assets/90144967/0c569856-b851-494a-bbd6-659a949dbc9c">

**Comment by ubuntuegor** [comment]
Thanks for submitting the issue! Did you try rebooting the PC, or restarting explorer.exe from the Task Manager? This is a [limitation of Windows](https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/integrate-packaged-app-with-file-explorer#support-generic-files-or-folders-and-perform-complex-tasks), unfortunately, item will appear only after one of these is done

**Comment by cleuton4k** [comment]
I tried everything but it doesn't appear

**Comment by ubuntuegor** [comment]
What types of files did you try this on? The item should only appear on popular formats like .mp4, .mkv, and so on

**Comment by ubuntuegor** [comment]
Yep, Win11 23H2, it's there. I'm out of ideas for why it doesn't appear for you, sorry.

<img width="315" alt="image" src="https://github.com/ubuntuegor/FluentInfo/assets/6959139/a75c57f3-39ed-48f7-aad6-97113dd5a252">

**Comment by realgooseman** [comment]
It didn't work for me as well on Windows 11, but after restarting the windows explorer the FluentInfo entry appeared in the context menu when right clicking media files (videos and images). Give it a try, maybe it will fix it.
